notebooks/search/03-ELSER.ipynb

```diff
@@ -12,7 +12,7 @@
     "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb)\n",
     "\n",
     "\n",
-    "Learn how to use the [ELSER](https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html) for text expansion-powered semantic search."
+    "Learn how to use the [ELSER v2](https://www.elastic.co/guide/en/machine-learning/master/ml-nlp-elser.html) for text expansion-powered semantic search."
```

Comment:
keep it as current as when 8.11 is released, this current page link will be updated.